docs/querying_history.rst

```diff
@@ -175,30 +175,49 @@ model history.
     <Poll: Poll object as of 2010-10-25 18:04:13.814128>
 
 
-Save without a historical record
---------------------------------
+Save Without Creating Historical Records
+----------------------------------------
 
-If you want to save a model without a historical record, you can use the following:
+If you want to save model objects without triggering the creation of any historical
+records, you can set a ``skip_history_when_saving`` attribute to ``True``
+on each object before saving - for example like this:
 
 .. code-block:: python
 
-    class Poll(models.Model):
-        question = models.CharField(max_length=200)
-        history = HistoricalRecords()
+    poll.skip_history_when_saving = True
+    poll.save()
+    # We recommend deleting the attribute afterward
+    del poll.skip_history_when_saving
 
-        def save_without_historical_record(self, *args, **kwargs):
-            self.skip_history_when_saving = True
-            try:
-                ret = self.save(*args, **kwargs)
-            finally:
-                del self.skip_history_when_saving
-            return ret
+This also works when creating an object
+(albeit only when calling the model's standard Python constructor followed by saving):
+
+.. code-block:: python
 
+    # Note that `Poll.objects.create()` is not called
```

Comment:
Maybe we put this after the code example?

> Historical records will always be created using the ``create`` manager method.

Reply:
Also I bet it's possible for us to manage this by overriding the ``create`` method on the manager class and accepting ``skip_history_when_saving`` as a kwarg.

Reply:
> Maybe we put this after the code example?
> > Historical records will always be created using the ``create`` manager method.

Sure, I can place it inside a `.. note::` :)

Reply:
> You think we should do that in this PR?

Definitely no :smile: A context manager sounds like a fantastic idea. Dang, that's clever!

Reply:
Alright :) I agree, though credits for the idea goes to https://github.com/jazzband/django-simple-history/issues/642#issuecomment-922110914 😄